### PR TITLE
Enhance result image

### PR DESCRIPTION
**Pull request by Perflyst**
Some small enhancements to the result image

--
Add information about what is what
Add used speedtest backend
Add timestamp

### CODE REVIEW

File: results/index.php [hunk 1]
```diff
@@ -138,6 +152,8 @@ function format($d){
 $jitMeterBbox=imageftbbox($FONT_METER_SIZE,0,$FONT_METER,$jit);
 $mbpsBbox=imageftbbox($FONT_MEASURE_SIZE_BIG,0,$FONT_MEASURE,$MBPS_TEXT);
 $msBbox=imageftbbox($FONT_MEASURE_SIZE,0,$FONT_MEASURE,$MS_TEXT);
+$serverBbox=imageftbbox($FONT_SERVER_SIZE,0,$FONT_SERVER,$server);
+$POSITION_X_SERVER=$WIDTH-$serverBbox[4]-4*$SCALE;
```
Comment (Perflyst) [inline results/index.php]: I am not sure if this is the correct position, it is the same as used above but at least for me it looks like it does not fit perfectly.

download

**Comment by adolfintel** [comment]
There are a couple of problems:
* The ISP and host text are very close to overlapping
* The extra string may contain other stuff that's not the host, if we want to do this, we need to separate this information in the DB

**Comment by Perflyst** [comment]
> The ISP and host texts are very close to overlapping

I can make the text smaller, though this risk will always be there. The only way to avoid is to not use too long names for backends or to remove un-relevant stuff (as we do on librespeed.org e.g. number and ISP is removed in result image).

> The extra string may contain other stuff that's not the host

What else could be contained? Also shouldn't `json_decode($server,true)["server"];` the json decoder be able to handle additional name/value pairs?

**Comment by adolfintel** [comment]
> I can make the text smaller, though this risk will always be there. The only way to avoid is to not use too long names for backends or to remove un-relevant stuff (as we would do on librespeed.org e.g. number and ISP is removed in result image).

It needs to be redesigned, we could make it more similar to ookla's. Let's discuss this on matrix

> What else could be contained? Also shouldn't `json_decode($server,true)["server"];` the json decoder be able to handle additional name/value pairs?

Potentially, anything that was passed as extra. The server name in the extra field was an afterthough, this needs to be addressed and the server ID needs to be added to entries in the database. We'll do this once we've figured out the IDs thing.

**Pull request closed without merging**